src/Detector/Adapter/Concrete5Adapter.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * @package    CMSScanner
```

Comment:
Could you please update the header of your files to make it match the header in the rest of the repo?